internal/kafka/internal/handlers/validation_test.go

```diff
@@ -79,10 +79,10 @@ func Test_Validation_validateKafkaClusterNameIsUnique(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			gomega.RegisterTestingT(t)
+			RegisterTestingT(t)
```

Comment:
This doesn't needs to be executed for each iteration. We can move it outside of the loop.

Reply:
Thanks @ziccardi I'll tidy that up in the tests in this file